packages/features/ee/organizations/components/AddNewTeamsForm.tsx

```diff
@@ -54,21 +55,42 @@ export const AddNewTeamsForm = () => {
     },
   });
 
+  const validateInput = () => {
+    if (inputValues.includes("")) {
+      showToast(t("team_names_empty"), "error");
+      return false;
+    }
+    const duplicates = inputValues.filter((item, index) => inputValues.indexOf(item) !== index);
+    if (duplicates.length) {
+      showToast(t("team_names_repeated"), "error");
+      return false;
+    }
+    return true;
+  };
+
+  const handleFormSubmit = () => {
+    if (validateInput()) {
+      createTeamsMutation.mutate({ orgId, teamNames: inputValues });
+    }
+  };
+
   return (
     <>
       {Array.from({ length: counter }, (_, index) => (
         <div className="relative" key={index}>
           <TextField
             key={index}
-            value={inputValues[index]}
+            value={inputValues[index] || ""}
```

Comment:
As inputValues can be undefined we set value on textField to empty string so react doesn't complain about
controlled/uncontrolled inputs.